src/bastion/Cargo.toml

```diff
@@ -43,7 +43,7 @@ unstable = ["bastion-executor/unstable"]
 bastion-executor = { version = "= 0.3.5-alpha", path = "../bastion-executor" }
 lightproc = { version = "= 0.3.5-alpha.0", path = "../lightproc" }
 
-dashmap = "3.4.0"
+lever = { path = "../../../lever" }
```

Comment:
Probably should be something like `lever = "0.1.0"` here instead of a path on the local machine?

Reply:
Need to release first. The latest changes are not released into crates.io. Initial PR that transforms us is this.